Packs/AzureKeyVault/Integrations/AzureKeyVault/AzureKeyVault.py

```diff
@@ -564,70 +624,91 @@ def create_or_update_key_vault_command(client: KeyVaultClient, args: Dict[str, A
     ignore_missing_vnet_service_endpoint = argToBoolean(
         args.get('ignore_missing_vnet_service_endpoint', True))
     ip_rules = argToList(args.get('ip_rules'))
-
-    response = client.create_or_update_key_vault_request(vault_name, object_id, location, sku_name, keys_permissions,
-                                                         secrets_permissions, certificates_permissions,
-                                                         storage_accounts_permissions, enabled_for_deployment,
-                                                         enabled_for_disk_encryption, enabled_for_template_deployment,
-                                                         default_action, bypass, vnet_subnet_id,
-                                                         ignore_missing_vnet_service_endpoint, ip_rules)
+    subscription_id = args.get('subscription_id', params.get('subscription_id'))
+    resource_group_list = argToList(args.get('resource_group_name', params.get('resource_group_name')))
+
+    all_responses = []
+    for single_resource_group in resource_group_list:
+        response = client.create_or_update_key_vault_request(subscription_id, single_resource_group,
+                                                             vault_name, object_id, location, sku_name, keys_permissions,
+                                                             secrets_permissions, certificates_permissions,
+                                                             storage_accounts_permissions, enabled_for_deployment,
+                                                             enabled_for_disk_encryption, enabled_for_template_deployment,
+                                                             default_action, bypass, vnet_subnet_id,
+                                                             ignore_missing_vnet_service_endpoint, ip_rules)
+
+        if response.get('error'):
```

Comment:
Here again, maybe we shouldn't fail the command if only one of the resources caused an error. Let's discuss that.

Reply:
It is impossible to return a green and red entry with command_results. Alternatively, it is possible to return str(error) instead of an exception. That's not a good idea, because if both values are incorrect, I want an exception.

Reply:
Not relevant anymore after the demo right?